airbyte-integrations/connectors/destination-chroma/setup.py

```diff
@@ -6,7 +6,7 @@
 from setuptools import find_packages, setup
 
 MAIN_REQUIREMENTS = [
-    "airbyte-cdk[vector-db-based]",
+    "airbyte-cdk[vector-db-based]==0.51.22",
```

Comment:
Do we need to pin exact? If we want to at least include this version but go up to 0.51.x we could use `~=`

Reply:
As the vector db CDK currently changes dramatically even in patch versions, I would like to keep it like this for now until things are more stable-